jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/Main.java

```diff
@@ -104,33 +104,35 @@ public static Map<String, Object> shipData(Map<String, String> params) {
             errorList.add(errorMap);
             return ErrorManageUtil.formErrorCode(jobName, errorList);
         }
-        String ds = params.get("datasource");
-        String overrideSourcePath = params.get(Constants.OVERRIDE_SOURCE_PATH);
+
+       String ds = params.get("datasource");
+        //reverting these changes as it is not required due to mapper changes
+     /*   String overrideSourcePath = params.get(Constants.OVERRIDE_SOURCE_PATH);
         LOGGER.debug("overrideSourcePath:{}", overrideSourcePath);
-        try {
+     */   try {
             AWSCredentialProvider awsCredentialProvider = new AWSCredentialProvider();
             dataSource = params.get("datasource");
             LOGGER.debug("dataSource:{}", dataSource);
-            //overrideSourcePath will sent by Mapper Lambda to indicate the path where shipper should exactly pick from
+          /*  //overrideSourcePath will sent by Mapper Lambda to indicate the path where shipper should exactly pick from
             if (null != overrideSourcePath && !overrideSourcePath.isEmpty()) {
                 srcFolder = overrideSourcePath;
             }
             //Aws & Azure will pick up from s3.data location
-            else {
+            else {*/
                 srcFolder = params.get("s3.data");
-            }
+            //}
             LOGGER.debug("srcFolder:{}", srcFolder);
             ESManager.configureIndexAndTypes(ds, errorList);
             errorList.addAll(new EntityManager().uploadEntityData(ds, srcFolder));
             new ExternalPolicies().uploadPolicyDefinition(ds);
             errorList.addAll(new AssetGroupStatsCollector().collectAssetGroupStats());
             errorList.addAll(new IssueCountManager().populateViolationsCount());
             errorList.addAll(new AssetsCountManager().populateAssetCount());
-             //As part of new Plugin Development , backup files will be handled by Shipper Batch Job.Hence Collector responsibility lies only with Collecting Data.
-            LOGGER.info("Back Up logic code is going to be executed");
-            LOGGER.debug("Invoking/Calling doBackUpAndCleanUpInventory() method");
-            doBackUpAndCleanUpInventory(dataSource, srcFolder, awsCredentialProvider);
-            LOGGER.info("Execution of doBackUpAndCleanUpInventory() method is done");
+             //Reverting changes since this part goes to  Collector again.
```

Comment:
Please, let's not leave commented out code that is absolute, let's just remove it, which makes PR also simpler to review.

Reply:
Raised another PR to address this.
https://github.com/PaladinCloud/CE/pull/1619